src/applications/health-care-supply-reordering-redirect/routes.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+// Force a redirect since React Router's Redirect does not reload the page.
+class CustomRedirect extends React.Component {
+  constructor(props) {
+    super(props);
+    window.location.replace(props.url);
+  }
+}
```

Comment:
The React docs say that `render()` is a required method for class-based components, as well as saying that functional components are recommended over class-based components: https://react.dev/reference/react/Component

Reply:
Interesting. I updated it and makes it a bit simpler.